### DISCUSSION TITLE
Support to macOS Sequoia

### DISCUSSION THREAD

**Discussion by semedilino-073** [d99kris/nchat#305] [General]
Hi,
I just want the developer to know that you've done a really good job, this really blew me away. Anyway, I read that the developer only tested macOS Sonoma's compatibility and I don't know if I'm supposed to say it, but it works in macOS Sequoia too. At least, it launches and it links with my Whatsapp account. I didn't install Telegram. Maybe you'll have to do some tests and if you need any sort of help or something like that, I want you to know that I'm here. I'm not an expert, but, you know, that's interesting to me. Also, some emojis don't show up. I don't actually know if that's intended or not and if that is in the Sonoma version too. I also noticed that the emojis have a different name and maybe the emoji menu can be made in a better way. Like, is it possible to build a program that searches the emoji in the same way that the macOS menu, opened with ctrl+command+space, does?

**Comment by d99kris** [comment]
Hi @semedilino-073 - thanks for the positive feedback 👍 

W.r.t. the documented supported platforms, the purpose is mainly to indicate what platforms I perform testing on. Generally speaking nchat is expected to work on most modern macOS versions (say from within a few years), and similar for Linux. But I cannot test all possible platforms, so I just list what I have tested.

For emoji selector, it's a little tricky. One can use the macOS emoji selector, but all emojis will not render correctly in Terminal. So the emoji selector in nchat has intentionally left several ones out, that do not render properly in Terminal. Once/if Terminal fixes it, nchat will likely enable them again.
